astronomer/providers/google/cloud/sensors/gcs.py

```diff
@@ -39,32 +40,18 @@ class GCSObjectExistenceSensorAsync(BaseOperator):
         account from the list granting this role to the originating account (templated).
     """
 
-    template_fields = ("bucket", "object", "impersonation_chain")
-    ui_color = "#f0eee4"
-
     def __init__(
         self,
-        *,
-        bucket: str,
-        object: str,  # noqa: A002
```

Comment:
why are we removing these? Have you tested all these sensors and ensured that the functionality remains as earlier?

Reply:
Yes, I have run all the sensors dags and this variable is present in `GCSObjectExistenceSensor`  which is being inherited instead of `BaseOperator`.